cpmpy/transformations/linearize.py

```diff
@@ -445,6 +435,63 @@ def only_positive_bv(lst_of_expr):
 
     return newlist
 
+def only_positive_bv_sub(expr):
+    """
+        Replaces a linear expression containing NegBoolView with an equivalent expression using only BoolVar.
+        expr is expected to be a var, sum or wsum.
+        
+        If the expression has a NegBoolView, the function returns a tuple of the new expression as a wsum and a constant, 
+        which can be added to the wsum or subtracted from the rhs (depending on the usecase).
```

Comment:
this sentence still unclear... it just returns a (wsum, tuple) constant, no need to specify what one 'could' do with it (e.g. the two ways in which you used it); that is not relevant, just what it returns.

Could also be more formal with

Arguments:
  expr:  Expression, variable, sum or wsum
  
  Returns:
    (wsum, const)  where wsum is a wsum where all bools are BooLVar (no negboolviews), and const is a constant value that must be added to the expression (or removed from the rhs of a comparison, as fit)
    
  (in correct rst style)
  
  note that I did end up saying smth about what can be done with it, but now it follows from the meaning and role of the constant: if you ignore it you are not equivalent!